openbas-api/src/main/java/io/openbas/migration/V3_33_ADD_Index_inject.java

```diff
@@ -0,0 +1,22 @@
+package io.openbas.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Statement;
+
+@Component
+public class V3_33__Add_index extends BaseJavaMigration {
+
+    @Override
+    public void migrate(Context context) throws Exception {
+        Statement select = context.getConnection().createStatement();
+        select.execute(""
+            + "CREATE INDEX idx_inject_inject_injector_contract on injects (inject_injector_contract);"
+            + "CREATE INDEX idx_injector_contract_injector on injectors_contracts (injector_id);"
+            + "CREATE INDEX idx_injector_inject on injectors (injector_id);"
+            + ""
```

Comment:
The last one is already present (pk). I think we should generally add indexes on foreign keys.

Reply:
Fix